src/features/movie/Movie.tsx

```diff
@@ -1,12 +1,16 @@
-import { movie } from "root/data";
+import { movie } from "fakeData/data";
 import { MovieHeader } from "./components/movieHeader/MovieHeader";
 import { Overview } from "./components/overview/Overview";
+import { Cast } from "./components/cast/Cast";
+import { Social } from "./components/social/Social";
 
 export const Movie = () => {
   return (
     <div key={movie.id}>
       <MovieHeader movie={movie} />
       <Overview movie={movie} />
+      <Cast movie={movie} />
+      <Social movie={movie} />
```

Comment:
A pattern is emerging here where all components depend on a movie. It's not obvious from this usage to determine which data each of the components needs, and if the movie's structure ever changes, you will need to update all of these components. It will be easier to maintain to extract every parameter of the movie here and pass only the necessary data to each component. 

This brings a few benefits:
- forces you to think which data needs to actually be passed in, and usage will expose this fact
- makes code more modular, and less coupled, making it easier to maintain in the future